src/whisk.rs

```diff
@@ -45,6 +45,13 @@ pub struct TrackerProof {
     s: Fr,
 }
 
+/// Convenience wrapper for whisk verifiers
+#[derive(Clone, Debug, CanonicalSerialize, CanonicalDeserialize)]
+pub struct ShuffleProof {
+    m: G1Projective,
```

Comment:
In the rest of the codebase we call this `M` to exactly match the notation of the paper and also use `#![allow(non_snake_case)]` so that cargo doesn't shout at us. I would prefer it that way, but no strong opinion if you don't like it.